build.gradle

```diff
@@ -35,6 +35,7 @@ ext {
 	description = 'RADAR Rest Source Authorizer handles authorization for data access from third party APIs for wearable devices or other connected sources.'
 
 	okhttp3Version = '3.13.1'
+	managementPortalVersion = '0.5.7'
```

Comment:
You can go for `MP 0.5.8` since it is already available.

Reply:
Done